src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -86,6 +88,9 @@ class NewConsentGroup extends Component {
       }
     );
   }
+  componentWillUnmount() {
+    spinnerService._unregisterAll();
+  }
```

Comment:
This is a good example of why the global state approach is so problematic. Imagine that there is a parent component surrounding `NewConsentGroup` that may have other components in it. One of those other components has an active spinner going for some other, unrelated reason. This code will effectively preempt the other component's use of the spinner and now you have introduced a bug.

Reply:
Thanks for pointing out this, I've used named spinners for `NewConsentGroup` and `LinkWizard` components. Always knowing this will be removed in [BTRX-754](https://broadinstitute.atlassian.net/browse/BTRX-754).